cmake/SetupCompilerFlags.cmake

```diff
@@ -4,6 +4,9 @@
 #
 # SPDX-License-Identifier: (MIT)
 ##############################################################################
+message(STATUS "Setting C standard to 99")
+set(CMAKE_C_STANDARD 99)
```

Comment:
If you write
```C++
if(NOT DEFINED CMAKE_C_STANDARD)
    message(STATUS "Setting C standard to 99") 
    set(CMAKE_C_STANDARD 99) 
endif()
```
users can overwrite this setting with their own value from the command-line, e.g. to build for C11.

Reply:
See #266